protocol/protos/WifiCore.proto

```diff
@@ -89,3 +89,13 @@ message AccessPointConfiguration
 {
     Dot11PhyType PhyType = 1;
 }
+
+message AccessPointCapabilities

```

Comment:
Should this also contain the various dot11 algos as fields?

Reply:
I think so, but I'm not totally certain what mix of things will need to go here. For example, do both the algorithms _and_ the ciphers need to go here? What about the phy type? 

I initially decided to defer this decision for another PR so I could think it through more, but now that you've raised it, I don't see any case where we wouldn't provide the algorithms, so, I'll add a field here for that.

Reply:
Added.